Anytype/Sources/PresentationLayer/Common/SwiftUI/SyncStatus/Model/Optional+SyncStatusInfo.swift

```diff
@@ -29,7 +29,7 @@ extension Optional where Wrapped == SyncStatusInfo {
         }
     }
     
-    var networkIconColor: Color {
+    var networkIconColor: SyncStatusInfo.NetworkIconBackground {
```

Comment:
Extension for `extension Optional where Wrapped == SyncStatusInfo` requires  to sync types, methods, and mades a "hidden" logic. If you make a one default `SyncStatusInfo`,  won't it be more obvious?

Reply:
Fixed in this pr
https://github.com/anyproto/anytype-swift/pull/1781/files#diff-d850f457cfe080deb43b173bba206055a58fd9fa7e6b9cebe1f2fd2081607520